src/Table.luna

```diff
@@ -181,10 +488,38 @@ class Table:
     def shortRep:
         "Table<" + self.columnCount.toText + "×" + self.rowCount.toText + ">"
 
+    # Compute pairwise Pearson's correlation of columns, excluding NA/null values.
+    #
+    # > import Dataframes.Column 
+    # > import Dataframes.Types
+    # > import Dataframes.Table 
+    # > l1 = [1,2,3,,5]
+    # > l2 = [11,,13,14,15]
+    # > col1 = Column.fromList "col1" Int64Type l1
+    # > col2 = Column.fromList "col2" StringType l2
+    # > table = Table.fromColumns [col1 , col2]
+    # > correlations = table.corr
+    #
+    # `return`: `Table` value with pairwise correlations calculated for `self` table.
```

Comment:
What are the names of columns in `return`?

Reply:
the self table

Reply:
Not resolved here